bin/automated

```diff
@@ -6,8 +6,6 @@
 set -e
 
 # Variables
-SOURCE_BRANCH="master"
-ORIGIN_URL=`git config --get remote.origin.url`
 ORIGIN_CREDENTIALS=${ORIGIN_URL/\/\/github.com/\/\/$GH_TOKEN@github.com}
 COMMIT_MESSAGE=$(git log -1 --pretty=%B)
 TALKS="infra-2017 machine-learning-2017 front-2017"
```

Comment:
1. Please, try to limit the input on this PR to changes related to it, otherwise you pollute the revision with _side quests_.
2. That list is edited by other bin/script and was thought in case we wanted to push a new talk without having to publish it right away.

Thank you!